res/schema.xml

```diff
@@ -584,4 +584,20 @@ reapplying those migrations.
       UPDATE library SET filetype='aiff' WHERE filetype='aif';
     </sql>
   </revision>
+  <revision version="40" min_compatible="3">
+    <description>
+      Add table for track relations.
+    </description>
+    <sql>
+      CREATE TABLE IF NOT EXISTS relations (
+        rl_id INTEGER PRIMARY KEY AUTOINCREMENT,
+        track_a INTEGER NOT NULL REFERENCES library(id),
+        track_b INTEGER NOT NULL REFERENCES library(id),
```

Comment:
is one the predecessor to other the successor? Maybe we can mark them like this.